CONTRIBUTING.md

```diff
@@ -5,24 +5,20 @@ If you'd like to get involved in the TSC or contact them then the best way is to
  1. Sign up to the [AdoptOpenJDK Slack](https://adoptopenjdk.net/slack.html)
  1. Drop a message in the _general_ channel
 
-# How to Contribute to the TSC Knowledge Base
+## How to Contribute to the TSC Knowledge Base
 
-The TSC Knowledge Base is designed to be a gateway (or index if you will) into the various repositories that make up the 
-AdoptOpenJDK build farm.  Each repository is responsible for maintaining its _own_ issues, pull requests and 
-documentation, the TSC repo simply provides a starting point.
+The TSC Knowledge Base is designed to be a gateway (or index if you will) into the various repositories that make up the AdoptOpenJDK build farm.  Each repository is responsible for maintaining its _own_ issues, pull requests and documentation, the TSC repo simply provides a starting point.
 
-For the TSC, we use the the repository [Project](https://github.com/AdoptOpenJDK/TSC/projects/1) to manage Issues and Pull 
-Requests.  Although straight out Pull Requests are welcome, we prefer that you:
+For the TSC, we use the the repository [Project](https://github.com/AdoptOpenJDK/TSC/projects/1) to manage Issues and Pull Requests.  Although straight out Pull Requests are welcome, we prefer that you:
 
  1. [Review existing issues](https://github.com/AdoptOpenJDK/TSC/issues) to avoid duplication
- 1. [Create a new Issue](https://github.com/AdoptOpenJDK/TSC/issues/new)
-    1. Please add the TSC project and add a label
+ 1. [Create a new Issue](https://github.com/AdoptOpenJDK/TSC/issues/new) - Please add the TSC project and add a label
```

Comment:
I'm not sure that people without additional permissions in the organisation are able to add to the project and add a label so this may be frustrating for true newbies

Reply:
Fixed.